platformcomponents/desktop/messages.json

```diff
@@ -72,6 +72,36 @@
           "text": "@theme-text-warning-normal"
         }
       }
+    },
+    "badge": {
+      "regular": {
+        "#normal": {
+          "background": "@theme-background-secondary-normal",
+          "text": "@theme-text-secondary-normal"
+        },
+        "#hovered": {
+          "background": "@theme-background-secondary-hover",
+          "text": "@theme-text-primary-normal"
+        },
+        "#active": {
+          "background": "@theme-background-secondary-active",
+          "text": "@theme-text-primary-normal"
+        }
+      },
+      "important": {
```

Comment:
same as the other one, this is called "pinning" according to the figma. 
![image](https://user-images.githubusercontent.com/56999622/192315238-e108108b-69a1-4836-a4cf-3c79b7b8268f.png)